sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -647,9 +647,9 @@ def get_backoff_time(self, settings):
 class StorageBearerTokenCredentialPolicy(BearerTokenCredentialPolicy):
     """ Custom Bearer token credential policy for following Storage Bearer challenges """
 
-    def __init__(self, credential, **kwargs):
+    def __init__(self, credential, *, audience: str = STORAGE_OAUTH_SCOPE, **kwargs):
```

Comment:
I would just make audience a required param with no default value (to match Core). You'll also want to update the type hint below.